### DISCUSSION TITLE
Double slash in URL

### DISCUSSION THREAD

**Discussion by TRIBEmediaDK** [elementor/elementor#18099] [Developers Community]
Hi 
Elementor Support sent us here, we have been having a recurring issue on our site, where double slash is being inserted into the url for a elementor .js file. 

https://www.tribemedia.dk/wp-content/plugins/elementor-pro/assets//lib/instant-page/instant-page.min.js?ver=3.6.3

Do you know why this would happen or if there is something we can do to fix it?

Thanks in advance, 
Nicolai Jensen

Webdeveloper
Tribe Media

**Comment by arsanet** [comment]
I had this problem not only on one site but on several sites that I had. Is there any news about this? Does anyone have a solution and was able to solve it? Those who had this problem, please write under this post or give a simple like so that the statistics are clearer and more accurate.

**Comment by mkarim1378** [comment]
yes I have this problem now and also before. Elementor team should fix this tiny bug.

**Comment by Divergentech** [comment]
I've had this in two pages for 1+ year. Since last update its starting to show on several more.

**Comment by LantrixUK** [comment]
I have this issue also. The issue is with: /wp-content/plugins/elementor-pro/assets//lib/instant-page/instant-page.min.js?ver=3.21.0 You can see the double-slash between "assets" and "lib"

**Comment by marcdumont-za** [comment]
The issue is explained and solved here -  https://marcdumont.com/elementor-users-fix-double-slash-url-errors-with-ease/
